src/core/Tile/Tile.test.tsx

```diff
@@ -12,27 +12,21 @@ import { MenuItem } from '../Menu';
 import SvgPlaceholder from '@itwin/itwinui-icons-react/cjs/icons/Placeholder';
 
 it('should render in its most basic state', () => {
-  const { container } = render(
-    <Tile name='test-name' thumbnail={<SvgPlaceholder />} />,
-  );
+  const { container } = render(<Tile name='test-name' />);
   expect(container.querySelector('.iui-tile')).toBeTruthy();
 
-  expect(container.querySelector('.iui-thumbnail')).toBeTruthy();
+  expect(container.querySelector('.iui-thumbnail')).toBeFalsy();
```

Comment:
I think you changed the only test for this class..

Reply:
Huh? We have 3 tests for thumbnail

Reply:
hmm.. maybe browser search failed for `.iui-thumbnail` then...